### ISSUE TITLE
[xmr] Error(s) refreshing network stats: One or more errors occurred.

### ISSUE THREAD

**Issue by fathusband** [coinfoundry/miningcore#420]
### Description

been running xmr pool for almost 3 days  but recently I could see some error message from pool log 

### Steps to Reproduce

1 - Setup XMR main chain 
2 - Connect Clients on MiningCore.
3 - Wait for blocks

### Expected behavior

no error showing in pool log

### Actual behavior

[What actually happens]

not sure whats happening here, 

### Reproduces how often

this error will show up every 2 - 3  hour 

[What percentage of the time does it reproduce?]

### Version

v 3.0
### Additional Information

##error log:

[2018-10-07 18:17:26.0362] [W] [xmr] Error(s) refreshing network stats: One or more errors occurred. (Unexpected character encountered while parsing value: <. Path '', line 0, position 0.) (Code -500)
[2018-10-07 18:17:26.0362] [E] [xmr] System.NullReferenceException: Object reference not set to an instance of an object.
   at MiningCore.Blockchain.Monero.MoneroJobManager.UpdateNetworkStatsAsync() in /userA/miningcore/src/MiningCore/Blockchain/Monero/MoneroJobManager.cs:line 186 System.NullReferenceException: Object reference not set to an instance of an object.
   at MiningCore.Blockchain.Monero.MoneroJobManager.UpdateNetworkStatsAsync() in /userA/miningcore/src/MiningCore/Blockchain/Monero/MoneroJobManager.cs:line 186    at MiningCore.Blockchain.Monero.MoneroJobManager.UpdateNetworkStatsAsync() in /userA/miningcore/src/MiningCore/Blockchain/Monero/MoneroJobManager.cs:line 186

###xmr config

{
      "id": "xmr",
      "enabled": true,
      "coin": {
        "type": "XMR"
      },
      "hasLegacyDaemon": false,
      "enableInternalStratum": false,
      "externalStratums": [
        {
          "url": "tcp://192.168.1.3:6000",
          "topic": "Monero"
        }
      ],
      "address": "45P3tNGKfBTR2Kt7wtz9zHZrpt6Yg3RaDPpeF88TJYU14Q5sYBTHWucKPsCTpShRosgp7wyjzXHqZTRgaCuCPR6kHDgiQVz",
      "rewardRecipients": [
        {
          "address": "45P3tNGKfBTR2Kt7wtz9zHZrpt6Yg3RaDPpeF88TJYU14Q5sYBTHWucKPsCTpShRosgp7wyjzXHqZTRgaCuCPR6kHDgiQVz",
          "percentage": 0.75
        }
      ],
      "blockRefreshInterval": 500,
      "jobRebroadcastTimeout": 10,
      "clientConnectionTimeout": 600,
      "banning": {
        "enabled": true,
        "time": 600,
        "invalidPercent": 50,
        "checkThreshold": 50
      },
      "ports": {
        "3092": {
          "listenAddress": "0.0.0.0",
          "difficulty": 1,
          "name": "ASIC Mining",
          "varDiff": {
            "minDiff": 1,
            "targetTime": 15,
            "retargetTime": 90,
            "variancePercent": 30
          }
        }
      },
      "daemons": [
        {
          "host": "10.100.1.6",
          "port": 18081,
          "user": "user",
          "password": "password"
        },
        {
          "host": "10.100.1.6",
          "port": 18082,
          "user": "walletuser",
          "password": "password",
          "category": "wallet"
        }
      ],
      "paymentProcessing": {
        "enabled": true,
        "minimumPayment": 0.1,
        "payoutScheme": "PPLNS",
        "payoutSchemeConfig": {
          "factor": 2
        },
        "minersPayTxFees": true
      }
    },

**Comment by fathusband** [comment]
im actually using Latest miningcore  2018 from master . sorry for the typo

**Comment by oliverw** [comment]
miningcore 2018?

**Comment by fathusband** [comment]
yes the one that i clone from master (https://github.com/coinfoundry/miningcore)  few days  ago
